src/languages/cpp.js

```diff
@@ -94,6 +94,52 @@ export default function(hljs) {
     relevance: 0
   };
 
+  const OPERATORS = [
+    '+=',
+    '-=',
+    '*=',
+    '/=',
+    '%=',
+    '&=',
+    '|=',
+    '^=',
+    '<<=',
+    '>>=',
+    '++',
+    '--',
+    '<<',
+    '>>',
+    '&&',
+    '||',
+    '<=>',
+    '==',
+    '!=',
+    '<=',
+    '>=',
+    '~',
+    '!',
+    '%',
+    '^',
+    '&',
+    '*',
+    '<',
+    '(?<!-)>',
+    '/',
+    '-(?!>)',
+    '+',
+    '?',
+    '(?<!:):(?!:)',
```

Comment:
Do we have a test case for these?  I'm pretty sure they won't work a these are string literals... `regex.escape` is going to escape all the regex stuff... we'd need to make these raw regex and then teach `escape` to just return raw regex as-is...

We should probably need a new `markup/cpp/operators.txt` test case with very minimal examples of every operator, just stuff like:

```
a == b
b > c
d * f
```

Or i'd even settle for just the full list of operators, one per line.

If there are false positives our regex are working around we'd want those included also - to make sure the behavior is correct.

---

Sadly we can't use negative look-behind until version 12 as it would be a breaking change.  What you can often do in these cases is match the offending pattern with a mode, just not assign it a scope... this effectively "skips over" the content and prevents it from being matches by other rules.